fluid/voice.cpp

```diff
@@ -1805,14 +1805,15 @@ void Sample::optimize()
       signed short peak;
       float normalized_amplitude_during_loop;
       double result;
+      int i;
 
       /* ignore ROM and other(?) invalid samples */
       if (!s->valid())
             return;
 
       if (!s->amplitude_that_reaches_noise_floor_is_valid) { /* Only once */
             /* Scan the loop */
-            for (size_t i = s->loopstart; i < s->loopend; i++) {
+            for (i = (int)s->loopstart; i < (int) s->loopend; i ++) {
```

Comment:
These changes don't seem to belong to this PR?

Reply:
and even if, shouldn't we steer away form old C style casts?
The spaces between `(int)` and `s->` and between `i` and `++` are not wanted either
And if `loopstart` and `loopend` return `size_t`, that's the type to use anyhow

Reply:
@Jojo-Schmitz These are accidentally changes, I got confused in git rebase and accidentally rolled back other people's changes. Basically I fixed it everywhere, but then I missed it, thanks to @dmitrio95  for noticing.